.storybook/themes/news.scss

```diff
@@ -1607,6 +1629,23 @@
 						),
 					),
 				),
+				xl-promo: (
+					components: (
+						heading: (
+							align-self: center,
+							font-size: var(--global-font-size-15),
+							justify-content: center,
+							line-height: 1.2,
```

Comment:
At Desktop this uses Heading Size 1. Should this use `var(--global-line-height-3)`?
Or are we using this until the global line-height tokens are multipliers?

Reply:
wait, what's happening here? we're not using the css variables because they're off -- can we fix them?

Reply:
using pixel values

Reply:
updated